### ISSUE TITLE
Captcha is not accessible

### ISSUE THREAD

**Issue by jsmaniac** [schneidmaster/gitreports.com#115]
Hello,

The captcha is not accessible to blind or ill-sighted people.

A better (but not perfect) option would be to rely on Google's reCAPTCHA, which includes an audio "fallback".

It would be nice to have the option to generate a simple project-specific token (just a random string of characters), and include it in the URL to skip the captcha altogether. That way, developers can include a "submit a bug report" button in their application, knowing that mass-harvesting tokens would involve understanding the source code, and therefore be a rather difficult task for bots.

**Comment by schneidmaster** [comment]
Hi there. I agree this would be a great improvement and I'd be happy to merge a PR. Unfortunately my own schedule is fairly busy at the time (and, as you probably know, Git Reports is a free service) so I can't make any promises about when I'll have time to work on it myself.

**Comment by LeifAndersen** [comment]
Would you be opposed to using a service like reCaptcha? (We're happy to make the actual change to use it.)

**Comment by schneidmaster** [comment]
I'd have no problems with using recaptcha :)

**Comment by schneidmaster** [comment]
One thing to keep in mind is locale support (this might have been why I didn't use recaptcha the first time around -- can't remember). Git Reports has French and Polish users who have contributed locale translations of the site content; simple_captcha allows you to customize all of its messages with locales (see [here](https://github.com/schneidmaster/gitreports.com/blob/master/config/locales/en.yml#L18-L24) for example). So ideally the new CAPTCHA should also translate appropriately based on locale header. I assume recaptcha does this since Google obviously has a huge international user base but I'm not sure offhand.

**Comment by CareyJWilliams** [comment]
I can't find an explicit mention of support, but there's a comprehensive languages code page which includes both French and Polish in v2: https://developers.google.com/recaptcha/docs/language

I'd be happy to look into this, if there's still room for a PR.

**Comment by schneidmaster** [comment]
Yeah, feel free to send a PR. The folks who opened the issue never got around to sending me one 🙂

**Comment by LeifAndersen** [comment]
Ya, sorry. We hand't gotten around to it yet. Feel free to do it, otherwise I will eventually get around to it. :)

Either way though, thank you.
